### ISSUE TITLE
TEAL - post-compile resource usage stats

### ISSUE THREAD

**Issue by jeapostrophe** [algorand/go-algorand#989]
When `goal clerk compile` finishes, I'd like it to give me some data about how close I am to the resource limits; in particular, I'm worried about the program size. (My programs are getting pretty large, but still compile. See https://github.com/reach-sh/reach-lang/blob/master/examples/rps/build/rps_while.teal for my ~1000 line TEAL program.)

**Comment by algobolson** [comment]
Good idea. That'd be useful. `goal clerk compile` emits compiled bytes without header, so `ls -l` or `wc -c` will give you the simple length of that.

**Comment by jannotti** [comment]
I'm happy to add this, as I'm currently adding checks to complain about large or costly programs anyway. I think they should only print this info with an extra flag, as I suspect scripts often grab the hash from the current output.

On they other hand, with v4, cost checks won't be off much value, so is reporting the size of the output very interesting?

**Comment by jeapostrophe** [comment]
Yea, I'm not super interested in this anymore, as I'm using the `algod` compiler API endpoint and I've expanded by interfacing library to check this stuff for me, so I don't really care. But, back when I first started using TEAL, I really wanted to know this.
